PolyEngine/Core/Src/Logger.hpp

```diff
@@ -40,18 +44,43 @@ namespace Poly
 		const T& head, Args&&... tail) {
 		constexpr char marker[] = "{}";
 		std::string::size_type format_marker_pos = fmt.find(marker);
-		stream << fmt.substr(0, format_marker_pos);
+		stream << String(fmt.substr(0, format_marker_pos).c_str());
 		if (format_marker_pos != fmt.npos) {
 			stream << head;
 			sprint(stream, level, fmt.substr(format_marker_pos + sizeof(marker) - 1),
 				tail...);
 		}
 		else
-			stream << std::endl;
+			stream << String("\n");
 	}
 
 	class CORE_DLLEXPORT Console : public BaseObject<> {
 	public:
+		Console() : Ostream(new std::ostream(std::cout.rdbuf())){} // this is a hack :v
```

Comment:
This is not hack. remove this comment. Also, codding style, put space before first bracket.